### PR TITLE
Added check if doryd container already exists before deployment

### PR DESCRIPTION
**Pull request by bhagyashree-sarawate**
Hi sneha,
Please review the code changes for the issue 740
Attached the output.
[740_output_with_fix.txt](https://github.com/hpe-storage/python-hpedockerplugin/files/3684421/740_output_with_fix.txt)

Regards
Bhagyashree Sarawate

### CODE REVIEW

**Comment by sneharai4** [comment]
Fixes #740

File: ansible_3par_docker_plugin/tasks/configure_doryd_service.yml [hunk 1]
```diff
@@ -31,22 +34,34 @@
       k8s_1_13 : true
     when: oc_result.stderr !="" and ks8_result is defined and ks8_result.stdout is search("1.13") and ks8_result.stderr == ""
 
+    - name: Check whether doryd as a container exists for multimaster Kubernetis/OS setup
+    shell:
+      kubectl get pods --namespace kube-system -o wide | grep doryd
+    register: result
+    ignore_errors: yes
+    when: multimaster == true and (os_3_11 == true or k8s_1_13 == true)
```
Comment (sneharai4) [inline ansible_3par_docker_plugin/tasks/configure_doryd_service.yml]: Fix the indentation here.

File: ansible_3par_docker_plugin/tasks/configure_doryd_service.yml [hunk 2]
```diff
@@ -31,22 +34,34 @@
       k8s_1_13 : true
     when: oc_result.stderr !="" and ks8_result is defined and ks8_result.stdout is search("1.13") and ks8_result.stderr == ""
 
+    - name: Check whether doryd as a container exists for multimaster Kubernetis/OS setup
```
[the remark this answers is not in the document]
Comment (bhagyashree-sarawate) [reply ansible_3par_docker_plugin/tasks/configure_doryd_service.yml]: done

File: ansible_3par_docker_plugin/tasks/configure_doryd_service.yml (hunk 1 above)
Comment (bhagyashree-sarawate) [reply ansible_3par_docker_plugin/tasks/configure_doryd_service.yml]: done

**Pull request merged**